intranet/static/js/eighth/ui_init.js

```diff
@@ -1,72 +1,80 @@
 /* global $ */
 $(function() {
-
     selectizeIDScore = function(search) {
         var ths = $(this)[0];
         var score = this.getScoreFunction(search);
         return function(item) {
-            if(parseInt(item.value) == parseInt(ths.lastQuery)) {
-                return 10000;
-            }
-            return score(item);
+            return (parseInt(item.value) === parseInt(ths.lastQuery)) ? 10000 : score(item);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should specify the radix of the mentioned string in parseInt calls.](https://www.codacy.com/app/tjcsl/ion/file/2755451119/issues/source?bid=3176160&fileBranchId=3355598#l7)